pkg/sdk/go/certs.go

```diff
@@ -20,33 +20,38 @@ const (
 
 // Cert represents certs data.
 type Cert struct {
-	ThingID    string    `json:"thing_id,omitempty"`
-	CertSerial string    `json:"cert_serial,omitempty"`
-	ClientKey  string    `json:"client_key,omitempty"`
-	ClientCert string    `json:"client_cert,omitempty"`
-	Expiration time.Time `json:"expiration,omitempty"`
+	SerialNumber string    `json:"serial_number,omitempty"`
+	Certificate  string    `json:"certificate,omitempty"`
+	Key          string    `json:"key,omitempty"`
+	Revoked      bool      `json:"revoked,omitempty"`
+	ExpiryDate   time.Time `json:"expiry_date,omitempty"`
```

Comment:
This is not a date, please change it to `Expiry` or `ExpiryTime`. The same goes for JSON tag name.

Reply:
Update this in the rest of the codebase and API.